test/system/helpers.bash

```diff
@@ -405,6 +405,19 @@ function _ensure_pod_state() {
     is "$output" "$2" "unexpected pod state"
 }
 
+# Wait for the container's (1st arg) running state (2nd arg)
+function _ensure_container_running() {
+    for i in {0..5}; do
+        run_podman container inspect $1 --format "{{.State.Running}}"
+        if [[ $output == "$2" ]]; then
+            break
```

Comment:
The best part of functions is that you can simply `return`. Then the fallthrough can simply be `die "Timed out waiting for .State.Running on $1 to be $2`.

But it's not worth a repush, this is just something for next time. LGTM.

Reply:
Thanks for explaining!  Looks like I need to ramp up my bash skills a bit further.